src/components/toolbox/Chart/chart.stories.js

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import StoryWrapper from '../../../../.storybook/components/StoryWrapper/StoryWrapper';
+import Box from '../box';
+import BoxHeader from '../box/header';
+import Chart from './index';
+import { styles } from '../../../utils/chartOptions';
+import style from './chart.stories.css';
+
+storiesOf('Toolbox', module)
+  .add('Chart', () => (
+    <StoryWrapper>
+      <Box className={style.wrapper}>
+        <BoxHeader><h1>Line Chart</h1></BoxHeader>
+        <div className={style.container}>
+          <Chart
+            type="line"
+            data={{
+              labels: ['Jan', 'Feb', 'Mar', 'Apr', 'May', 'Jun', 'Jul', 'Oug', 'Sep', 'Oct', 'Nov', 'Dec'],
+              datasets: [
+                {
+                  label: '123456789L',
+                  data: [35, 55, 23, 67, 98, 46, 134, 66, 70, 33, 100, 120],
+                  backgroundColor: styles.whiteSmoke,
+                  borderColor: styles.borderColor,
+                  pointBorderColor: styles.borderColor,
+                  pointBackgroundColor: styles.whiteColor,
+                  pointHoverBackgroundColor: styles.whiteColor,
+                  pointHoverBorderColor: styles.ultramarineBlue,
+                  pointHoverBorderWidth: 4,
+                  borderWidth: 2,
```

Comment:
IMO the point of creating our toolbox Chart is, that you can call it without any style properties and it looks like Lisk. Of course, any styles can be changed in a specific page, but the default values should IMO be from Lisk style guide. Currently, removing all the styles in this file makes it look like this (not looking like Lisk):
![Screenshot 2019-11-18 at 08 57 03](https://user-images.githubusercontent.com/1254342/69034534-be1bb580-09e1-11ea-8877-1a7cab859ec4.png)